runtime-modules/content-working-group/Cargo.toml

```diff
@@ -0,0 +1,114 @@
+[package]
+name = 'substrate-content-working-group-module'
+version = '1.0.0'
+authors = ['Joystream contributors']
+edition = '2018'
+
+[features]
+default = ['std']
+std = [
+	'sr-primitives/std',
+	'srml-support/std',
+	'system/std',
+    'serde',
+    'codec/std',
+    'primitives/std',
+    'rstd/std',
+    'membership/std',
+    'forum/std',
+    'hiring/std',
+    'stake/std',
+    'minting/std',
+    'versioned_store/std',
+    'versioned_store_permissions/std',
+    'recurringrewards/std',
+]
+
+
+[dependencies.sr-primitives]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'sr-primitives'
+rev = 'c37bb08535c49a12320af7facfd555ce05cce2e8'
+
+[dependencies.srml-support]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'srml-support'
+rev = 'c37bb08535c49a12320af7facfd555ce05cce2e8'
+
+[dependencies.system]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'srml-system'
+rev = 'c37bb08535c49a12320af7facfd555ce05cce2e8'
+
+[dependencies.rstd]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'sr-std'
+rev = 'c37bb08535c49a12320af7facfd555ce05cce2e8'
+
+[dependencies.serde]
+features = ['derive']
+optional = true
+version = '1.0.101'
+
+[dependencies.primitives]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'substrate-primitives'
+rev = 'c37bb08535c49a12320af7facfd555ce05cce2e8'
+
+[dependencies.codec]
+default-features = false
+features = ['derive']
+package = 'parity-scale-codec'
+version = '1.0.0'
+
+[dependencies.forum]
```

Comment:
I think dependency on forum is only for use of the validation method, `InputValidationLengthConstraint` that should probably be moved to the common crate later.

Reply:
https://github.com/Joystream/substrate-runtime-joystream/issues/184